cmd/peggo/orchestrator.go

```diff
@@ -19,7 +19,7 @@ import (
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc"
 
-	umeedpfconfig "github.com/umee-network/umee/price-feeder/config"
+	umeedpfprovider "github.com/umee-network/umee/price-feeder/oracle/provider"
```

Comment:
```suggestion
	"github.com/umee-network/umee/price-feeder/oracle/provider"
```

Reply:
we already have a pakcage called provider inside peggo

Reply:
oh, the ethereum/provider? Then how about calling this one `oprovider` or `oracleprovider`?